home/models.py

```diff
@@ -300,6 +300,36 @@ def clean(self, value):
         return result
 
 
+class SMSBlock(blocks.StructBlock):
+    message = blocks.TextBlock(
+        help_text="each message cannot exceed 160 characters.",
+        validators=(MaxLengthValidator(160),),
+    )
```

Comment:
I think keep it this way for now, but one thing to think of is that we can have multi-part smses, where the message gets split and sent (and charged) as multiple messages, but then the phone reassembles them on the other side and displays it as a single message. Previously I think we've not used this to keep cost low, but just to keep it in mind. I think we can create another block type to handle that though

Reply:
Cool, understood and agreed